src/core/models.py

```diff
@@ -32,6 +32,7 @@
 from submission import models as submission_models
 from utils.logger import get_logger
 from utils import logic as utils_logic
+from core.janeway_global_settings import SUMMERNOTE_SENTINEL
```

Comment:
@joemull this is a django antipattern: The settings object is not a module, but rather a lazy loaded singleton. In order to ensure it is only loaded once and only when expected, it needs to be imported as `from django.conf import settings` (https://docs.djangoproject.com/en/1.11/topics/settings/#using-settings-in-python-code)

Janeway always loads the `janeway_global_settings.py` module when the `settings` object is first loaded so you can then call `settings.SUMMERNOTE_SENTINEL` even for those using a custom settings file.

All that said, I think it might be simpler to store the SUMMERNOTE_SENTINEL as a constant either on `core.forms` or `core.models`

Reply:
Got it--thanks for catching this. I moved it to `core.models`.